impl/src/main/java/org/ehcache/impl/internal/statistics/DefaultTierStatistics.java

```diff
@@ -73,7 +73,7 @@ public DefaultTierStatistics(Cache<?, ?> cache, String tierName) {
     conditionalRemove = findOperationStatistic(cache, tierName, "conditionalRemove");
     eviction = findOperationStatistic(cache, tierName, "tier", "eviction");
     expiration = findOperationStatistic(cache, tierName, "expiration");
-    compute = findOperationStatistic(cache, tierName, "compute");
+    compute = findOperationStatistic(cache, tierName, "getAndCompute");
```

Comment:
Isn't both compute and getAndCompute required?

Reply:
getAndCompute is just another flavor of compute, so only compute is needed, will correct it